hw/ip/otbn/doc/_index.md

```diff
@@ -623,13 +638,62 @@ Before being stored in SRAM, the instruction word with the attached Integrity Pr
 The scrambling is reversed on a read.
 
 The ephemeral memory scrambling key and the nonce are provided by the [OTP block]({{<relref "/hw/ip/otp_ctrl/doc" >}}).
-They are set once when OTBN block is reset, and changed whenever a secure wipe of the instruction memory is performed.
-(TODO: Link to it once we have specified secure wipe.)
+They are set once when OTBN block is reset, and changed whenever a [secure wipe]({{<relref "#design-details-secure-wipe">}}) of the instruction memory is performed.
 
 The Integrity Protection Code is checked on every memory read, even though the code remains attached to the data.
 A further check must be performed when the data is consumed.
 Detected integrity violations in the data memory raise a fatal `imem_error`.
 
+### Secure Wipe {#design-details-secure-wipe}
+
+In the following situations OTBN itself initiates a full secure wipe:
+* The lifecycle controller asks for it through its escalation signal.
+* A fatal alert is issued. 
+  In this case, a full secure wipe is performed as local action.
+
+Software can trigger a full state wipe by writing 3’b111 to SEC_WIPE (i.e. set all individual state wipe bits).
+
+#### Data Memory (DMEM) Secure Wipe
+
+The wiping is performed by securely replacing the memory scrambling key, making all data stored in the memory unusable. 
+The key replacement is a two-step process:
+
+* Overwrite the 128b key of the memory scrambling primitive with randomness from URND.
+  This action is deterministic in time and takes a single cycle.
+* Request new scrambling parameters from OTP. 
+  The request takes multiple cycles to complete.
+
+Host software can initiate a data secure wipe by writing 1 to the SEC_WIPE.DMEM register field.
```

Comment:
I think you mean "data memory secure wipe" here? But, as I said above, I think this would be much clearer if we called it something like "DMEM key rotation".